processor/src/coin/mod.rs

```diff
@@ -3,82 +3,83 @@ use std::{marker::Send, sync::Arc};
 use async_trait::async_trait;
 use thiserror::Error;
 
+use frost::{curve::Curve, sign::PreprocessMachine, FrostKeys};
 use transcript::RecommendedTranscript;
-use frost::{curve::Curve, FrostKeys, sign::PreprocessMachine};
 
+pub mod ethereum;
 pub mod monero;
 pub use self::monero::Monero;
 
 #[derive(Clone, Error, Debug)]
 pub enum CoinError {
-  #[error("failed to connect to coin daemon")]
-  ConnectionError
+    #[error("failed to connect to coin daemon")]
```

Comment:
2-space ident, please, not to mention the reformatting. While I'll agree to cargo fmt to resolve these debates, that should be done in a followup PR.

Reply:
reverted all auto-fmt'd files